### DISCUSSION TITLE
Request For NEO 2 User Guide

### DISCUSSION THREAD

**Discussion by gstrib5** [WLAN-Pi/feedback#33] [General Inquiries] [answered]
Hello! I'm trying to find the user guide for the NEO 2... Can someone direct me to where I might find this? (It does not seem to be on the main user guide site)

**Comment by jiribrejcha** [comment, marked as answer]
It lives here:
https://wlan-pi.github.io/wlanpi-documentation/

**Reply by gstrib5** [reply]
Thank you!!!
